src/tests/unit/tests/DetailsView/components/adhoc-static-test-view.test.tsx

```diff
@@ -96,7 +96,7 @@ describe('AdhocStaticTestView', () => {
         ];
 
         it.each(scenarios)('handles %s', (_, content, guidance) => {
-            props.deps = 'stub-deps' as AdhocStaticTestViewDeps;
+            props.deps = 'stub-deps' as unknown as AdhocStaticTestViewDeps;
```

Comment:
This seems a bit odd. Why not just use an empty object or something (easier to cast) of that sort here?

Reply:
It's for the purposes of snapshot readability; I think seeing `{ deps: 'stub-deps', ... }` in a snapshot makes it obvious that the stub deps have been piped around correctly and unmodified, vs `{ deps: {}, ... }` being less obvious about the deps being piped correctly